client/src/components/pages/Message.tsx

```diff
@@ -488,6 +490,48 @@ const MessageScreen: FC = () => {
     params: {channelId},
   } = useRoute<RouteProp<MainStackParamList, 'Message'>>();
 
+  // Delete notification if user is foreground in channel.
+  useEffect(() => {
+    const subscription = Notifications.addNotificationReceivedListener(
+      (notification) => {
+        const parsedNotificationData = JSON.parse(
+          notification.request.content.data.data as string,
+        );
```

Comment:
compared to other similar codes like `L517-520` in this file and [L131-L134 in MainStackNavigator](https://github.com/dooboolab/hackatalk/blob/master/client/src/components/navigations/MainStackNavigator/index.tsx#L131-L134), this has only different style. do you have any special reason?

I think this kind of code style has to become the same.

Reply:
I changed like you commented, but I think it went back to original. I'll fix it.

Reply:
> I changed like you commented, but I think it went back to original. I'll fix it.

I checked your commit, but no change was detected. please check if you'd pushed the commit to your repo.

and your other code in the same file [L517-L529](https://github.com/SimYunSup/hackatalk/blob/feature/delete_same_notification/client/src/components/pages/Message.tsx#L517-L529) looks similar to this code block. so could you tell me why do you think it went back to original? and also I don't know what is the original 😆.

please understand me if I ask the dumb question 🙏